share/sysbuild/cmake/modules/sysbuild_extensions.cmake

```diff
@@ -321,16 +342,25 @@ function(ExternalZephyrProject_Cmake)
          "${ZCMAKE_APPLICATION}_BOARD:STRING=${${ZCMAKE_APPLICATION}_BOARD}\n"
     )
   endif()
+  list(APPEND sysbuild_cache_strings
+       "${ZCMAKE_APPLICATION}_KCONFIG_BINARY_DIR:PATH=${${ZCMAKE_APPLICATION}_KCONFIG_BINARY_DIR}\n"
+  )
 
   file(WRITE ${${ZCMAKE_APPLICATION}_CACHE_FILE}.tmp ${sysbuild_cache_strings})
   zephyr_file_copy(${${ZCMAKE_APPLICATION}_CACHE_FILE}.tmp
                    ${${ZCMAKE_APPLICATION}_CACHE_FILE} ONLY_IF_DIFFERENT
   )
 
+  set(dotconfigsysbuild ${${ZCMAKE_APPLICATION}_KCONFIG_BINARY_DIR}/.config.sysbuild)
```

Comment:
nit: I expected to find `.config.sysbuild` this next to `${app}/zephyr/.config`, not in `${app}/Kconfig`.

Reply:
Good point.
Other Kconfig generated files usually goes to `Kconfig` dir, but I agree, the name `.config.sysbuild` could indicate to users that this file ought to be located next to `.config`.

I will relocate, but allow others to give their feedback before doing so.

Reply:
Moved to `${app}/zephyr/.config.sysbuild`